frontend/.env.production

```diff
@@ -0,0 +1,17 @@
+# !! Don't put secrets in this file. Use .env.local for secrets instead !!
+# This file is checked into your git repo and provides defaults for non-sensitive
+# env vars when running `next dev` only.
+# Learn more: https://nextjs.org/docs/app/building-your-application/configuring/environment-variables
+
+# This is used to determin if this is a prod or a non prod environment for deployed environments. Currently used for noindex and google tag manager logic
+NEXT_PUBLIC_ENVIRONMENT=dev
```

Comment:
Shouldn't this be prod?

Reply:
yep good catch! just updated

>I'm not sure I understand what these files are for. Are they only used locally, and not used in our actual production deploys?

You can browse the [next docs](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#default-environment-variables) - but basically when you run the frontend locally with `npm run dev` it will load the `.env.development` vars (the browser only sees the `NEXT_PUBLIC_` prefixed ones for safety). 


Then when you run `npm run build` and `npm start` - it will load the `env.production` vars . The result here is dev and staging will point to prod API - I'm assuming that's ok but lmk if not. (I haven't actually hooked up the frontend to the api yet - that's the next PR after this)


I created a follow up ticket to come back for `dev` and `staging` https://github.com/HHS/simpler-grants-gov/issues/1385

Reply:
I'm also wondering if people think we should just store in AWS - I believe the `Sendy` API key is stored there?